src/frontend/src/showcase.ts

```diff
@@ -144,45 +149,34 @@ const iiPages: Record<string, () => void> = {
       cancel: () => console.log("canceled"),
       continue: (alias) => console.log("device alias:", alias),
     }),
-  authenticate: () =>
```

Comment:
The screenshot job did not delete `authenticate.png`. I thought it does also create commits for screenshots that got removed?

Reply:
I think I've removed that when I split the mobile vs desktop uploads; I'll remove the screenshot manually. Good catch 👌